geppetto-showcase/src/components/showcase/ShowcaseUtils.js

```diff
@@ -70,12 +69,12 @@ function getReactElement(dom) {
  * @param dom
  */
 
-function getProps(dom) {
+function getProps (dom) {
   const path = dom.getElementsByClassName('language-element')[0].innerHTML;
   // TODO: What happens to this path if we use geppetto-client package instead of local file?
```

Comment:
Is that TODO still open after this changed?

Reply:
No. Nice catch :smile: